near-contract-standards/src/non_fungible_token/payout/payout_impl.rs

```diff
@@ -0,0 +1,75 @@
+use super::NonFungibleTokenPayout;
+use crate::non_fungible_token::core::NonFungibleTokenCore;
+use crate::non_fungible_token::payout::*;
+use crate::non_fungible_token::NonFungibleToken;
+use near_sdk::{assert_one_yocto, env};
+use near_sdk::{require, AccountId, Balance, IntoStorageKey};
+use std::collections::HashMap;
+impl Royalties {
```

Comment:
nit: please add a space between the `impl` and the imports and between the methods in the `impl`.